### PR TITLE
[VFX] Added Kenzu Skills effects

### PR DESCRIPTION
**Pull request by saugon**
Based on: https://github.com/lambdaclass/champions_of_mirra/pull/1823

## Motivation

I’ve added Kenzu Skill1 basic attack effects, the Ultimate effect, and 4 Dash effects, along with a test scene and a cheat panel for testing. The request contains test code in PinnedEffectsController.cs that needs to be removed before merging.

## Summary of changes

Added Kenzu Skill1 basic attack effects, Ultimate effect and 4 Dash effects, test scene and cheat panel for testing.

## How has this been tested?

Effects could be tested via cheat panel in test scene or via generic gameplay

## Checklist
- [ ] I have tested the changes locally.
- [ ] I have tested the whole game after applying the changes, not only the affected areas.
- [ ] I self-reviewed the changes on GitHub, line by line.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

File: client/Assets/Scripts/Effects/PinnedEffectsController.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@ public class PinnedEffectsController : MonoBehaviour
 
     private void Start()
     {
+        pinned_effects_manager = FindFirstObjectByType<PinnedEffectsManager>();// FOR TEST CASES ONLY! REMOVE BEFORE MERGE
```
Comment (tkz00) [inline client/Assets/Scripts/Effects/PinnedEffectsController.cs:15]: Should we remove this?

**Review by tkz00 — CHANGES_REQUESTED** [review]
Also I'm not a fan of the shield VFX for Kenzu's ultimate skill, is that something to be taken into account here?

Screenshot_20240725_111324_client

File: client/Assets/Scripts/Effects/PinnedEffectsController.cs (hunk 1 above)
Comment (tkz00) [reply client/Assets/Scripts/Effects/PinnedEffectsController.cs:15]: Based on the description of the PR

Comment (saugon) [reply client/Assets/Scripts/Effects/PinnedEffectsController.cs:15]: Yes, definitely!

Comment (tkz00) [inline client/Assets/Scripts/Effects/PinnedEffectsController.cs:15]:
```suggestion
        
```

Comment (tkz00) [reply client/Assets/Scripts/Effects/PinnedEffectsController.cs]: Great!

**Review by tkz00 — APPROVED** [review]
Great! The effects look amazing and the test scene works great on my android.

**Review by AngieDutra — CHANGES_REQUESTED** [review]
Tried restarting Unity but I think this bug is here in this PR:
I can't die with muflus in the game because of the following
Screenshot 2024-07-25 at 17 27 07
Screenshot 2024-07-25 at 17 27 15
Screenshot 2024-07-25 at 17 32 59
All characters have lost their pick up item effect reference and their death feedback reference

**Review by AngieDutra — APPROVED** [review]
Nice ✅ 
I want kenzu in the game asap! He looks cool 🐱

**Review by tkz00 — APPROVED** [review]

**Pull request merged**